.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -160,6 +160,7 @@ jobs:
       run: mkdir -p definitions/${{ env.SCANNER_BUNDLE_VERSION }}
 
     - name: Run Updater (single bundle)
+      if: ${{ env.SCANNER_BUNDLE_VERSION != 'dev' }}
```

Comment:
We don't need this for 4.6 onward as well.

Reply:
Not sure this is resolved. I meant we just need to run this for `4.4.*` and `4.5.*` (which uses `v1`), but, so using `!= 'dev'` would include `v1`, `v2`, ...